skins/moono-lexicon/mainui.css

```diff
@@ -207,3 +207,10 @@ Special outer level classes used in this file:
 .cke_editable:focus:after {
 	border-color: #80acff; /* $primary-l1 */
 }
+
+/* Page editor inline editor specific style for reseting backgorund color and padding
+*/
+.page-editor__fragment-content .cke_editable {
```

Comment:
Styles specific to page editor should be in the page editor module, not in this repo.

Reply:
Exactly, the idea is try to fix this as part of the general skin. If we can't because it's not a bug but an specific behavior that differs from what we want to provide with the skin, then yes, this should be in portal. That's what we need to analyze.